### ISSUE TITLE
XMLDSig verification seems to be timing out

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-android#1593]
Maybe really slow?

**Comment by hboon** [comment]
It's happening again.

```
$ curl https://aw.app/api/v1/verifyXMLDSig
curl: (7) Failed to connect to aw.app port 443: Operation timed out
```

**Comment by colourful-land** [comment]
@James-Sangalli  Is it normal that aw.app resolves to 54.67.102.112 which has:

````
$ host 54.67.102.112
 ec2-54-67-102-112.us-west-1.compute.amazonaws.com. 

````

 I would expect it to resolve to AWS API Gateway, not to an ec2 host? Did you redirect this from an EC2 host?

**Comment by colourful-land** [comment]
> @James-Sangalli Is it normal that aw.app resolves to 54.67.102.112 which has:
> 
> ```
> $ host 54.67.102.112
>  ec2-54-67-102-112.us-west-1.compute.amazonaws.com. 
> ```
> 
> I would expect it to resolve to AWS API Gateway, not to an ec2 host? Did you redirect this from an EC2 host?

Called up @James-Sangalli he explained that it shouldn't resolve to API Gateway or lambda because it is a Spring service.

I provided a lambda implementation and a demonstraction to call it from Android & assigned it to someone (don't remember Sangalli or Zaki) earlier to make mobile apps directly call lambda - wasn't aware this hasn't been implemented for about a year, so the service is still with EC2.

It seems the service should be routed from ELB which might have changed the IP address. The correct way to do it is to link the domain (in route53) to the ELB balancer, but it seems to have been linked to the IP address of the balancer, which is prone to change. AWS might have changed the IP address.

(In layman term, it is like when you program MySQL app, copying value from another DB table instead of linking to the record in that table, so it breaks when the table is updated).
